mattermost/platform
# PLT-7185: Double clicking Create Account attempts to create an account twice

#### Summary
Disabled submit button after it is clicked for the first time, unless an error occurs.
Testing instructions:
- Log in to the server and use the Get Team Invite Link function to copy an invite link to your clipboard
- Open an Incognito window and paste the team invite link into the URL bar
- Once the page loads, open the Chrome developer console and throttle the network speed to better see the impact of the change
- Fill in the signup form and press the Submit button. Notice that it is disabled immediately on click, and stays disabled until you are redirected to the team that you have joined

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-7185

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
